testing/makefile

```diff
@@ -272,12 +287,32 @@ test_testcase:
 
 test_readme_commands:
 # Test readme commands work right after instantiation of test case container
-	cd .. && TESTCASE=testcase2 docker-compose up -d
+	cd .. && docker compose build -q web worker provision	
+	cd .. && docker compose up -d --quiet-pull web worker provision
+	python sleep10.py
+	curl http://127.0.0.1:80/measurements
+	curl http://127.0.0.1:80/forecast
+	curl http://127.0.0.1:80/advance -d '{"oveTSetRooHea_u":293.15,"oveTSetRooHea_activate":1, "oveTSetRooCoo_activate":1,"oveTSetRooCoo_u":298.15}' -H "Content-Type: application/json"
+	cd .. && docker compose down
+
+test_python2:
```

Comment:
I don't think we're still testing Python 2. We can remove this already.

Reply:
See: https://github.com/ibpsa/project1-boptest/issues/634.

Reply:
Agreed, but hm, this should have been removed in https://github.com/ibpsa/project1-boptest/pull/642 and subsequent master merge.  But removed now in https://github.com/ibpsa/project1-boptest/pull/652/commits/aedb62293e671e12fe152bb07429a96e3bd2189e.